### DISCUSSION TITLE
Eclipse Apoapsis / ORT Server Logo

### DISCUSSION THREAD

**Discussion by mnonnenmacher** [eclipse-apoapsis/ort-server#1066] [Show and tell]
A designer at Bosch has created drafts for the Logos for Eclipse Apoapsis and the ORT Server. I would like to collect some feedback from the community until the end of the week, so that we can incorporate it into the design.

![logos](https://github.com/user-attachments/assets/9c86e454-1be1-4464-b0a2-c7e8045e4db5)

The idea is to have similar logs for Eclipse Apoapsis and ORT Server, where the version for the ORT Server uses the color scheme of the ORT logo to show the relation of the two projects. The neutral version will also be used for the [guidance project](https://github.com/eclipse-apoapsis/guidance).

We also have a color palette:

![palette](https://github.com/user-attachments/assets/b9c88e13-00ec-4de9-9627-7037a22f4310)

And an example for how the Docusaurus page could look like:

![website](https://github.com/user-attachments/assets/d490a94e-074b-4f77-a55f-457484fa8f2d)

The Eclipse Foundation also has some guidelines for project logos:
https://www.eclipse.org/projects/handbook/#promoting-project-logo

**Comment by sschuberth** [comment]
I basically like this! Just the triangles in the "saturn-like" logos are too small and should be omitted IMO. Similarly, the shape of the sickle in one of the variants is too filigree; I would not use that variant of the logo.

**Comment by mmurto** [comment]
Awesome! Can you please post the color palette in a little higher resolution, so the color values are readable?

If I read the primary color right to be `#BE3450`, it [doesn't pass](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=BE3450) WCAG AAA accessibility contrast for normal size text in pure white (`#FFFFFF`), which is probably not ideal.

**Reply by mnonnenmacher** [reply]
I have updated the image.

**Comment by nnobelis** [comment]
I like the saturn-like ones: this is a reminder of the space context of the word "apoapsis".

**Comment by wkl3nk** [comment]
I think Logo V1 looks to much like a sickle or horns on a viking helmet. So I would prefer Logo V2 (Saturn design)

**Comment by mkurzman** [comment]
Supercool! I also vote for the rather "saturn-looking" design BUT this shall not be a saturn ring but the orbit around the ORT planet. Maybe this still could be fine-tuned to reduce the effect that everyone directly sees a saturn first.
BTW did you see in Detlef's slide that in SDV they also use galaxy-pictures ? So that fits a little bit to our story: https://openchain-project.github.io/Automotive-WG/slides/2024-09-10-Current%20EU%20approach%20to%20support%20the%20Open%20Source%20collaboration%20in%20the%20European%20Automotive%20Industry-Detlef-Zerfowski.pdf => slide 6
Maybe there is potential to develop this story further.

**Comment by mnonnenmacher** [comment]
Thank you all for the feedback, I have discussed it with the designer and will post an updated version of the logos and the color palette soon.

**Comment by mnonnenmacher** [comment]
Here are the updated Logos:

![Logo_colored](https://github.com/user-attachments/assets/eb0ca130-5c32-4038-a92c-f7f213415d23)
![Logo_neutral](https://github.com/user-attachments/assets/810a50e0-df57-416a-a989-8e02d3950486)

And the updated color palette (@mmurto it should now pass the accessibility checks):

![image](https://github.com/user-attachments/assets/a4bf5cc4-5e9c-4a30-bf01-aba4d368ad85)

**Comment by mnonnenmacher** [comment]
I am closing this discussion as we have the final versions of the logos, I will soon add them to the repository.
